2-repositories/iglu-server/README.md

```diff
@@ -39,8 +29,8 @@ $ docker run --name igluserver -p 8080 -d snowplow-docker-registry.bintray.io/sn
 
 ## New features
 
-1. `/api/debug` endpoint (when `debug = true` in configuration file)
-2. `/api/meta` enpoint TODO
+1. `/api/debug` endpoint (when `debug = true` in configuration file) that returns the Iglu Server's internal state, if in-memory storage is used instead of PostgreSQL. This should be used for development only
+2. `/api/meta` endpoint, including health check routes and a route that returns metadata about the Iglu Server (version, database type, schema count etc.)
 3. `patchesAllowed` setting, prohibiting patches in production servers and allowing in Mini
```

Comment:
Did we mention `dummy` mode? Also it has a hardcoded master `apikey`, which also should be mentioned somewhere.